dataactcore/scripts/pull_fpds_data.py

```diff
@@ -1477,16 +1481,21 @@ def get_data(contract_type, award_type, now, sess, sub_tier_list, county_by_name
 
     entries_processed = 0
     while True:
-        async def atom_async_get(entries_already_processed):
+        # pull in the next MAX_ENTRIES * REQUESTS_AT_ONCE until we get anything less than the MAX_ENTRIES
+        async def atom_async_get(entries_already_processed, total_expected_records):
             response_list = []
             loop = asyncio.get_event_loop()
+            requests_at_once = MAX_REQUESTS_AT_ONCE
+            if total_expected_records - entries_already_processed < (MAX_REQUESTS_AT_ONCE * MAX_ENTRIES):
```

Comment:
What if we expect 60 entries and we end up with 61 entries in the feed? Pulling more entries than expected protected us from that. This way, sure, if we get more than expected when it's not a round number it works, but if it's a multiple of 10 we miss things. Most often the "different amounts" are from us getting less records than expected but we can't rule out the opposite.

Reply:
Solution was adding an extra call to check for additional records.